astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -1,31 +1,37 @@
 import os
 from datetime import datetime, timedelta
+from typing import Any, List
 
 from airflow.models.dag import DAG
 from airflow.providers.amazon.aws.operators.s3 import (
     S3CreateBucketOperator,
+    S3CreateObjectOperator,
     S3DeleteBucketOperator,
 )
-from airflow.providers.amazon.aws.transfers.local_to_s3 import (
-    LocalFilesystemToS3Operator,
-)
 
 from astronomer.providers.amazon.aws.sensors.s3 import (
     S3KeySensorAsync,
-    S3KeySizeSensorAsync,
     S3KeysUnchangedSensorAsync,
-    S3PrefixSensorAsync,
 )
 
 S3_BUCKET_NAME = os.getenv("S3_BUCKET_NAME", "test-bucket-astronomer-providers")
-S3_BUCKET_KEY = os.getenv("S3_BUCKET_KEY", "test/example_s3_test_file.txt")
+S3_BUCKET_KEY = os.getenv("S3_BUCKET_KEY", "test")
+S3_BUCKET_KEY_2 = os.getenv("S3_BUCKET_KEY_2", "test2")
```

Comment:
S3_BUCKET_KEY_2 doesnt make much sense. What do we want to do here? Can you please explain

Reply:
This is been used to check a list of bucket keys accepted in `S3KeySensorAsync`.
Will change the name.